dom/examples/todo/src/filter.rs

```diff
@@ -49,7 +49,7 @@ pub fn filter_link(to_set: Visibility) -> Li {
     mox! {
         <li>
             <a style="cursor: pointer;"
-             class={if *visibility == to_set { "selected" } else { "" }}
+             class={if *visibility.commit_at_root() == to_set { "selected" } else { "" }}
```

Comment:
We shouldn't need to acquire a lock here to get the visibility for the revision, but I think this is a problem for you because of my past laziness.

At the top of this function is this attribute:

```
#[illicit::from_env(visibility: &Key<Visibility>)]
```

I think this needs to become 

```
#[illicit::from_env(visibility: &Commit<Visibility>, set_visibility: &Key<Visibility>)]
```

Which will require exposing the current commit via https://github.com/anp/moxie/blob/main/dom/examples/todo/src/lib.rs#L53 as well.